chirho/robust/ops.py

```diff
@@ -39,10 +39,21 @@ def influence_fn(
     target_params, func_target = make_functional_call(target)
 
     @functools.wraps(target)
-    def _fn(point: Point[T], *args: P.args, **kwargs: P.kwargs) -> S:
-        param_eif = linearized(point, *args, **kwargs)
-        return torch.func.jvp(
-            lambda p: func_target(p, *args, **kwargs), (target_params,), (param_eif,)
-        )[1]
+    def _fn(
+        points: Point[T],
+        pointwise_influence: bool = False,
```

Comment:
Can we move the new `pointwise_influence` keyword argument up to be a keyword argument of `linearize` and `influence_fn` instead of changing the signature of the output functions?

Reply:
Yup I can do that!